sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/SchemaRegistryClientLiveTests.cs

```diff
@@ -23,27 +24,43 @@ public SchemaRegistryClientLiveTests(bool isAsync, SchemaRegistryClientOptions.S
             _serviceVersion = version;
         }
 
-        private SchemaRegistryClient CreateClient(string format)
+        private string GetEndpoint(string format, int group)
```

Comment:
Ideally we'd be able to remove the group param and condition this on service version. But that is dependent on us getting to the point where we need one endpoint per format per service version, rather than what we have currently which appears to be 2.